src/views/article.vue

```diff
@@ -109,6 +120,20 @@ export default {
             }
             return style;
         },
+        sortTable(table, column) {
+            let revMlt = 1;
```

Comment:
What does ``revMlt`` stand for here?

Reply:
reverse multiplier (to reverse the order with a multiplier). I adjusted the name now 👍

Reply:
> ![image](https://user-images.githubusercontent.com/32354342/119651604-7a4f0b00-be25-11eb-8c65-8f3462828043.png)
> Looks a bit off with so much space on the right side of the table imo. Maybe this only happens on my wide-screen.

Adjusted it that table take full space within container

Reply:
Thanks for explaining.